examples/shared_storage/shared_storage.pro

```diff
@@ -0,0 +1,27 @@
+QT += quick
+
+# You can make your code fail to compile if it uses deprecated APIs.
+# In order to do so, uncomment the following line.
+#DEFINES += QT_DISABLE_DEPRECATED_BEFORE=0x060000    # disables all the APIs deprecated before Qt 6.0.0
+
+include($$PWD/../../shared_storage/shared_storage.pri)
+
+SOURCES += \
+        documentscontroller.cpp \
+        main.cpp
+
+RESOURCES += qml.qrc
+
+# Additional import path used to resolve QML modules in Qt Creator's code model
+QML_IMPORT_PATH =
+
+# Additional import path used to resolve QML modules just for Qt Quick Designer
+QML_DESIGNER_IMPORT_PATH =
+
+# Default rules for deployment.
+qnx: target.path = /tmp/$${TARGET}/bin
```

Comment:
I guess all of this template code can go, as this lib only supports Android? (Certainly not QNX)

Reply:
yup